packages/shared/components/index.js

```diff
@@ -8,7 +8,6 @@ export { default as Error } from './Error.svelte'
 export { default as Idle } from './Idle.svelte'
 export { default as Illustration } from './Illustration.svelte'
 export { default as Link } from './Link.svelte'
-export { default as LoadingScreen } from './LoadingScreen.svelte'
```

Comment:
This component got removed from there but its still present in the shared components folder, was the intention to move it somewhere else?

Reply:
during some conflict resolution, it was added again :pray: